File: laboratorna 3/src/Tablets.java
public class Tablets extends Goods {
    private int coundInBox;

    public Tablets(String name, double price, int expirationDate, CategoryType categoryType, int coundInBox) {
        super(name, price, expirationDate, categoryType);
        this.coundInBox = coundInBox;
    }

    public int getCoundInBox() {
        return coundInBox;
    }

    public void setCoundInBox(int coundInBox) {
        this.coundInBox = coundInBox;
    }

    @Override
    public String toString() {
        return "Tablets{" + "name='" + super.getName() + '\'' +
                ", price=" + super.getPrice() +
                ", expirationDate=" + getExpirationDate() +
                ", categoryType=" + getCategoryType() +
                "coundInBox=" + coundInBox +
                '}';
    }
}

File: lab-11 python/Tablets.py
from Goods import Goods

class Tablets(Goods):

    def __init__(self, name, price, expiration_date, category_type,cound_in_box):
        super(Tablets, self).__init__(name, price, expiration_date, category_type)
        self.cound_in_box = cound_in_box


def __str__(self):
    return super(Tablets, self).__str__() + ' volume = ' \
           + str(self.volume)